lib/coinbase/webhook.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A representation of a Webhook.
+  # This class provides methods to create, list, update, and delete webhooks
+  # that are used to receive notifications of specific events.
+  class Webhook
+    class << self
+      # Creates a new webhook for a specified network.
+      #
+      # @param network_id [String] The network ID for which the webhook is created.
+      # @param notification_uri [String] The URI where notifications should be sent.
+      # @param event_type [String] The type of event for the webhook. Must be one of the following:
+      #   - `WebhookEventType::UNSPECIFIED` ("unspecified")
+      #   - `WebhookEventType::ERC20_TRANSFER` ("erc20_transfer")
+      #   - `WebhookEventType::ERC721_TRANSFER` ("erc721_transfer")
+      # @param event_filters [Array<Hash>] Filters applied to the events that determine
+      #   which specific events trigger the webhook. Each filter should be a hash that
+      #   can include keys like `contract_address`, `from_address`, or `to_address`.
+      # @return [Coinbase::Webhook] A new instance of Webhook.
+      #
+      # @example Create a new webhook
+      #   webhook = Coinbase::Webhook.create(
+      #     network_id: 'ethereum_mainnet',
```

Comment:
```suggestion
      #     network_id: :ethereum_mainnet,
```

Note the difference between a symbol and string here! We will be cleaning this up a bunch on the next release!